fetch/api/redirect/redirect-count-cross-origin.js

```diff
@@ -0,0 +1,59 @@
+if (this.document === undefined) {
+  importScripts("/resources/testharness.js");
+  importScripts("../resources/utils.js");
+  importScripts("/common/utils.js");
+  importScripts("/common/get-host-info.sub.js");
+}
+
+function redirectCount(desc, redirectUrl, redirectLocation, redirectStatus, maxCount1, maxCount2, requirePreflight)
```

Comment:
redirect-count.py is expected to do that (is_final_redirection check or something like that)